third_party/license_info.csv

```diff
@@ -317,11 +317,11 @@ kubernetes-sigs/yaml,https://github.com/kubernetes-sigs/yaml/blob/master/LICENSE
 sourcegraph/go-diff,https://github.com/sourcegraph/go-diff/blob/master/LICENSE,MIT License,https://raw.githubusercontent.com/sourcegraph/go-diff/master/LICENSE
 sqs/pbtypes,https://github.com/sqs/pbtypes/blob/master/LICENSE,Apache License 2.0,https://raw.githubusercontent.com/sqs/pbtypes/master/LICENSE
 fvbommel/util,https://github.com/fvbommel/util/blob/master/LICENSE,MIT License,https://raw.githubusercontent.com/fvbommel/util/master/LICENSE
-bitbucket.org/bertimus9/systemstat,https://bitbucket.org/bertimus9/systemstat/src/master/LICENSE,MIT License,https://bitbucket.org/bertimus9/systemstat/src/master/LICENSE
-modernc.org/cc,https://gitlab.com/cznic/cc/blob/master/LICENSE,BSD 3-Clause "New" or "Revised" License,https://gitlab.com/cznic/cc/blob/master/LICENSE
-modernc.org/golex,https://gitlab.com/cznic/golex/blob/master/LICENSE,BSD 3-Clause "New" or "Revised" License,https://gitlab.com/cznic/golex/blob/master/LICENSE
-modernc.org/mathutil,https://gitlab.com/cznic/mathutil/blob/master/LICENSE,BSD 3-Clause "New" or "Revised" License,https://gitlab.com/cznic/mathutl/blob/master/LICENSE
-modernc.org/strutil,https://gitlab.com/cznic/strutil/blob/master/LICENSE,BSD 3-Clause "New" or "Revised" License,https://gitlab.com/cznic/strutil/blob/master/LICENSE
-modernc.org/xc,https://gitlab.com/cznic/xc/blob/master/LICENSE,BSD 3-Clause "New" or "Revised" License,https://gitlab.com/cznic/xc/blob/master/LICENSE
-heketi/utils,https://github.com/heketi/heketi/blob/master/LICENSE,Apache License 2.0,https://github.com/heketi/heketi/blob/master/LICENSE
-kr/logfmt,https://github.com/kr/logfmt/blob/master/Readme,MIT License,https://github.com/kr/logfmt/blob/master/Readme
+bitbucket.org/bertimus9/systemstat,https://bitbucket.org/bertimus9/systemstat/raw/master/LICENSE,MIT License,https://bitbucket.org/bertimus9/systemstat/raw/master/LICENSE
```

Comment:
This fix is already in this PR. Can we close PR #185 ?

Reply:
sure

Reply:
vis a vis @jlewi  comment on previous PR, these third party licenses are regenerated I believe, and hence the change is needed?

Reply:
This `license_info.csv` file has the url links to download the license texts. The current one contains wrong download links and hence the `make test` fails at license check. Getting this fixed the earlier the better I believe.